tests/sniffTest

```diff
@@ -68,15 +68,13 @@ if [[ $rc != 0 || -s err ]]; then
 	echo Output: $(cat out)
 	exit 1
 fi
-$RUNC kill c1
-rm -rf /run/opencontainer/containers/c1
```

Comment:
while eval "runc state c1 | grep -q 'destroyed'" sleep...
then delete.. could do that